templates/taxi/car_list.html

```diff
@@ -8,6 +9,10 @@ <h1>
     </a>
   </h1>
   {% if car_list %}
+    <form action="" method="GET" class="form-inline">
+      {{ search_form|crispy }}
+      <input type="submit" value="Search" class="btn btn-light">
+    </form>
```

Comment:
Move `search_form` to a separate file and include it where you need `search_from`